Open external links in new tab (#403)
VertaAI/modeldb: client/verta/docs/examples/examples.rst

Examples
========

> |GitHub repo| houses |Client workflows| for reference and exploration.

Author: ```suggestion
|Verta's GitHub repository| houses |various example projects| for reference and exploration.
```

- `Random Forest with Grid Search
  <https://github.com/VertaAI/modeldb/blob/master/client/workflows/examples/xgboost.ipynb>`__
- `Client Integration
  <https://github.com/VertaAI/modeldb/blob/master/client/workflows/examples/xgboost-integration.ipynb>`__

> .. |GitHub repo| raw:: html

Author: ```suggestion
.. |Verta's GitHub repository| raw:: html
```

  <a href="https://github.com/VertaAI/modeldb" target="_blank">Verta's GitHub repository</a>

> .. |Client workflows| raw:: html

Author: ```suggestion
.. |various example projects| raw:: html
```
